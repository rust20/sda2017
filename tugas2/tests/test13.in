15 6 8 5000 110 110
Arturia Pendragon;1;100;pedang
Amakusha;2;100;pedang
Nazuna;3;100;pedang
Feget;4;100;pedang
Emiye;25;100;pedang
Queen Riana;0;9999;pedang
Raven;45;12000;pedang
Rigel;40;30000;pedang
Stella;20;9999;pedang
Jyanu;22;100;pedang
Hector;1;11;pedang
Hercules;20;9999;pedang
Gilgamesh;1;9999;pedang
Cu;4;100;pedang
Kizuna;20;1000;pedang
1;110;Gilgamesh,Hercules
3;2;Arturia Pendragon,Amakusha,Feget
1;3;Nazuna,Emiye,Jyanu,Hector,Queen Riana
1;5;Rigel,Raven
109;110;Cu,Stella
107;1;Kizuna
1;107;11;1;panah;3
109;2;215;1;panah;3
2;1;40;13;pedang;4
110;1;201;300;pedang;1
110;110;10000;200;pedang;1000
63;54;10000;300;panah;400
49;59;900;1200;pedang;20
2;110;77;34;panah;20
..S#S.......................................#............................................................#D..S
D.##...........................................#.........................................................#...D
#S##.....................................................................................................#....
#.#.................................#....................................................................#....
#.#.......................................#.........................................#....................#....
#.#......................................................................................................#....
.........................................................................................................#....
...................................................................#.....................................#....
.........................................................................................................#....
.................#.......................................................................................#....
.........................................................................................................#....
.........................................................................................................#...#
..............................................................................................................
..............................................................................................................
.............................................................#................................................
..........................................#...................................................................
..............................................................................................................
.......................#......................................................................................
............................................................#........................#........................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
.....................................#........................................................................
.....................................#M#..................D...................................................
.......................................#......................................................................
.......................................#......................................................................
.......................................#......................................................................
.......................................#......................................................................
.......................................#......................................................................
.......................................#......................................................................
.......................................#......................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
.....................................................D........................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
..............................................................................................................
S.............................................................................................................
..............................................................................................................
.D...........................................................................................................S
D............................................................................................................D
