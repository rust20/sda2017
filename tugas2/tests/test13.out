1,107 RUN, kekuatan maksimal sejumlah: 0
1,110 Pahlawan yang ikut:Gilgamesh,Hercules
2,110 BATTLE, kekuatan: 9998, pahlawan: Gilgamesh,Hercules
109,110 Pahlawan yang ikut:Stella,Cu
110,110 BATTLE, kekuatan: 30097, pahlawan: Gilgamesh,Hercules,Stella,Cu
49,59 BATTLE, kekuatan: 30097, pahlawan: Gilgamesh,Hercules,Stella,Cu
63,54 BATTLE, kekuatan: 15047, pahlawan: Gilgamesh,Hercules,Stella,Cu
1,5 Pahlawan yang ikut:Rigel,Raven
3,2 Pahlawan yang ikut:Arturia Pendragon,Amakusha,Feget
1,3 Pahlawan yang ikut:Queen Riana,Nazuna,Jyanu,Emiye,Hector
2,1 BATTLE, kekuatan: 61998, pahlawan: Rigel,Raven,Gilgamesh,Hercules
109,2 BATTLE, kekuatan: 25999, pahlawan: Rigel,Raven,Gilgamesh
107,1 Pahlawan yang ikut:Kizuna
110,1 BATTLE, kekuatan: 30000, pahlawan: Rigel
Akhir petualangan Gudako
Level Gudako: 7224
Level pahlawan:
Gilgamesh: 1749
Hercules: 1748
Stella: 1701
Cu: 1701
Rigel: 315
Raven: 15
Arturia Pendragon: 1
Amakusha: 1
Feget: 1
Queen Riana: 1
Nazuna: 1
Jyanu: 1
Emiye: 1
Hector: 1
Kizuna: 1
